### ISSUE TITLE
complete documentation

### ISSUE THREAD

**Issue by PavelBlend** [freeglut/freeglut#156]
Where can I find complete documentation for `freeglut`? The documentation here is not complete: [API](https://freeglut.sourceforge.net/docs/api.php)

For example, the `glutPostRedisplay` function is not described at all. And many other functions also have no description.

I found this site with glut 3 documentation: [glut/spec3](https://www.opengl.org/resources/libraries/glut/spec3/node1.html). The last edits date back to 1996. Is there more up-to-date documentation?

**Comment by jtsiomb** [comment]
Yeah, our documentation is not complete. But it documents most of the freeglut-specific additions to the API. The original GLUT 3.x documentation is still perfectly valid for the rest of the API which behaves identical to GLUT.

Unfortunately we can't just merge the two together, without Mark Kilgard's approval to release his old documentation under a free licence. I'm not sure if anyone ever tried to contact him about this or not.

Ideally someone should fill in the gaps, but nobody stepped up all these years.

**Comment by drewwoods** [comment]
FWIW see #200 for proposals of improving the docs
